### PR TITLE
Added reporting guidelines to Code of Conduct

### PR DESCRIPTION
**Pull request by emilyacook**
Added detail about the Code of Conduct account, and reporting Code of Conduct violations within the Mattermost community server, on the Forum, and elsewhere.

Also made some slight categorization changes

### CODE REVIEW

File: contributors/contributors/contribution-guidelines.md [hunk 1]
```diff
@@ -64,17 +52,36 @@ Examples of representing our community include using an official e-mail address,
...
+
+## Reporting a Violation
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-community@mattermost.com.
+There are several ways you can report a Code of Conduct violation:
+* On the Mattermost community server: Send a direct message to @mattermostconduct
+  * If the violation is in a public or private channel, include a link to the conversation being reported.
+  * If the violation is in a direct message conversaion, provide screenshots.
```
Comment (emilyacook) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
  * If the violation is in a direct message conversation, provide screenshots.
```

**Review by jasonblais — DISMISSED** [review]
Great update :+1:

File: contributors/contributors/contribution-guidelines.md [hunk 2]
```diff
@@ -64,17 +52,43 @@ Examples of representing our community include using an official e-mail address,
...
-## Enforcement
+# Enforcement
+
+## Reporting a Violation
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-community@mattermost.com.
+There are several ways you can report a Code of Conduct violation:
+* On the Mattermost community server: Send a direct message to @mattermostconduct
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
* On the Mattermost Community server: Send a Direct Message to @mattermostconduct.
```

File: contributors/contributors/contribution-guidelines.md [hunk 3]
```diff
@@ -64,17 +52,43 @@ Examples of representing our community include using an official e-mail address,
...
+  * If the violation is in a public or private channel, include a link to the conversation being reported.
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
  * If the violation is in a Public or Private channel, include a link to the conversation being reported.
```

File: contributors/contributors/contribution-guidelines.md [hunk 4]
```diff
@@ -64,17 +52,43 @@ Examples of representing our community include using an official e-mail address,
...
+  * If the violation is in a direct message conversation, provide screenshots.
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
  * If the violation is in a Direct Message conversation, provide screenshots.
```

File: contributors/contributors/contribution-guidelines.md [hunk 5]
```diff
@@ -64,17 +52,43 @@ Examples of representing our community include using an official e-mail address,
...
+* On the Mattermost forum: Send a message to [@mattermostconduct](https://forum.mattermost.org/u/mattermostconduct/)
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
* On the Mattermost forum: Send a message to [@mattermostconduct](https://forum.mattermost.org/u/mattermostconduct/).
```

File: contributors/contributors/contribution-guidelines.md [hunk 6]
```diff
@@ -64,17 +52,43 @@ Examples of representing our community include using an official e-mail address,
...
+* Email conduct@mattermost.com
 
 All complaints will be reviewed and investigated promptly and fairly.
 
 All community leaders are obligated to respect the privacy and security of the
 reporter of any incident.
 
+**All reports will remain anonymous**. The exception is direct message violations: while we
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
**All reports will remain anonymous**. The exception is Direct Message violations: while we 
```

**Review by justinegeffen — DISMISSED** [review]
Some formatting and terminology suggestions, but no content changes. Approving in the interim. Thank you!

**Comment by emilyacook** [comment]
@justinegeffen Had to resolve a conflict due to this PR https://github.com/mattermost/mattermost-handbook/pull/730, can you review again so it can be merged?

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**